app_dart/test/src/utils.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
```

Comment:
Should we call this file mocks.dart inside a utilities folder?

Reply:
Done.